src/sentry/integrations/slack/service.py

```diff
@@ -190,7 +216,21 @@ def _get_notification_message_to_send(self, activity: Activity) -> str | None:
 
         Apparently the get_context is a very computation heavy call, so make sure to only call this once.
         """
-        notification_cls = EMAIL_CLASSES_BY_TYPE.get(activity.type, None)
+        try:
+            activity_type: ActivityType = ActivityType(activity.type)
+        except ValueError as err:
+            self._logger.info(
```

Comment:
worth using logger.exception instead so you don't have to stringify the error etc? and we didn't want to add a test for this case?

Reply:
`logger.exception` raises an error right? We want to _not_ raise an error for this expected scenario, but simply logging and returning a no-op

Reply:
Correct me if I'm wrong here, that would be very helpful in the future haha!

Reply:
https://docs.python.org/3/howto/logging.html#when-to-use-logging

Reply:
Ah sorry, I meant to say it creates/raises an error in Sentry, and we don't really need to capture this in Sentry

Reply:
got it. that makes sense